web/packages/test/scripts/electra-deploy-ethereum.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+set -eu
+
+source scripts/set-env.sh
+export output_electra_dir="/tmp/electra"
+
+start_geth() {
+    rm -rf $output_electra_dir
+    mkdir -p $output_electra_dir
+    mkdir -p $output_electra_dir/ethereum
+    cp config/genesis-electra.json $output_electra_dir
+    cp config/jwtsecret $output_electra_dir
+
+    echo "Starting geth local node"
+    docker run --rm \
+      -v "${output_electra_dir}:/mnt" \
+      docker.io/ethpandaops/geth:lightclient-prague-devnet-4 \
```

Comment:
I would suggest not introducing docker for the local setup, can maybe build from source if necessary like lodestar.

Reply:
Build geth from source: https://github.com/Snowfork/snowbridge/pull/1283/commits/23eb81d5bdf7951a725f2861831c4a5d0f9aa604